sycl/source/handler.cpp

```diff
@@ -404,6 +404,13 @@ void handler::processArg(void *Ptr, const detail::kernel_param_kind_t &Kind,
   }
 }
 
+// The argument can take up more space to store additional information about
+// MAccessRange, MMemoryRange, and MOffset added with addArgsForGlobalAccessor.
+// We use the worst-case estimate because the lifetime of the vector is short.
+// TODO: the constant can be removed if the size of MArgs will be calculated at
+// compile time.
+const size_t MaxNumAdditionalArgs = 12;
```

Comment:
Does it look better this way?
```suggestion
static constexpr size_t MaxNumAdditionalArgs = 12;
```

Reply:
I think - yes, thanks for the suggestion!